### DISCUSSION TITLE
Amazon Chronos Sales Forecasting Colab Notebook (with test datasets and input fields)

### DISCUSSION THREAD

**Discussion by smartaces** [amazon-science/chronos-forecasting#115] [Show and tell]
Hi Everyone, 

I am sharing this in case it is helpful, I created a **Google Colab Notebook** which enables testing of Chronos T5 small on small datasets for forecasting, all you need is a csv file with some data in to get started (or even if you don't have that I've provided some you can use). 

Colab Notebook: https://github.com/smartaces/amazon-chronos-t5-sales-forecasting

Video walkthrough of Notebook: https://www.youtube.com/watch?v=jyrOmIiI2Bc&t=103s

![image](https://github.com/amazon-science/chronos-forecasting/assets/108146501/de35489d-7712-4575-b9a8-f5ef18004f81)

I also created a series of inputs to make it easy to set forcast period lengths and intervals. 

The notebook also allows for comparison against actual datasets (via csv upload), so you can see how accurate the forecasts are. 

Basically this makes it super simple to test the awesomeness of Chronos. 

![image](https://github.com/amazon-science/chronos-forecasting/assets/108146501/d01c330b-70fe-4e31-b886-e01ff4c94a06)

![image](https://github.com/amazon-science/chronos-forecasting/assets/108146501/ccaafa41-2f92-4ddb-be9c-51aca02fba0e)

All outputs are exported as csv file as well. 

I am a big fan of this model, and have seen some really great results even with T5 small, hitting +90% accuracy. 

All the best

J

**Comment by abdulfatir** [comment]
Nice! Thanks for sharing.
